new structure for radworks docs hub

abbey-titcomb opened the pull request:
made a bunch of structural changes based on shelby's [figma](https://www.figma.com/file/zhG9dHQAONZv71X9TPI1xQ/DAO-wiki-%26-v2-Gov-Docs?node-id=0-1&t=fxPQNRdO4zRhSYZP-0) 

Fixes #226

joelhans reviewed (commented):
Hey @abbey-titcomb! Spent some time going through this. Overall, the structural changes seem sound—I can run this locally without any errors or major issues. A few things to point out that I couldn't comment on directly.

- In **Governance Overview**, the first two grid links (to **Get Started** and **Learn about the RadicleDAO**) are both broken because they've been renamed. Are those documents still going to be used after these structural changes? If not, we should remove those links.
- Maybe this is discussed beyond what I can see (I can't see the final copy doc atm), but I think there a missed opportunity on the **Mission & Vision** page, which is essentially the "homepage" for the wiki, to promote more of the DAO-centric content as opposed to always pushing readers to other parts of the ecosystem.
- We could also eliminate the current homepage (with the big **RADWorks Documentation** header) and send people directly into the wiki, rather than asking them to click again to get into the content. That homepage works better, in my opinion, where there's multiple distinct "paths" for documentation, like two products, but having one wiki is a lot simpler.

Some other comments dashed throughout. Sorry if I'm asking questions that have already been answered elsewhere!

shelbeeee commented on docs/community/get-started.md:
I think we want to keep this! Could you please fix the link?

joelhans replied:
I can fix it in a follow-up PR once this one is merged! I don't want to accidentally break anything by pushing into @abbey-titcomb's branch.

shelbeeee commented:
Re Governance Overview - all of those boxes (create a proposal, vote on a proposal, delegate votes) should all link out to specific parts of the governance manual (see yellow post-it on figma). 

Re Mission & Vision page - we should think about the intent of this page a bit more, but we also want to avoid duplicating the main DAO website which aims to achieve exactly that. 

Re homepage - maybe I am not seeing it on the code, but if you look at the figma I believe that was the plan anyways. Will let Abbey weigh in here.

abbey-titcomb commented on docs/community/02_our_story.md:
```diff
@@ -0,0 +1,106 @@
+title: Our Story
+---
+
+:::note
+
+This is a modified fork of the [Contributor’s Covenant](https://www.contributor-covenant.org/) — an open-source code of conduct created by [Coraline Ada Ehmke](https://where.coraline.codes/) and stewarded by the [Organization for Ethical Source](https://ethicalsource.dev/).
+
+:::
```
Just to port some copy into the page! Will update the pages with the first draft of copy now.

joelhans replied:
Awesome! I figured there was something new coming, but PRs are the best place to double-check...

joelhans commented:
@shelbeeee Got it—all those things make sense. Seems like most everything will be fixed/addressed when the copy changes are brought through after the fact.

abbey-titcomb commented:
@joelhans, thanks for the suggestions! Comments below:

- I've removed the broken grid links in **Governance Overview** for now. I think we'll update this when we dive into the copy a bit more.
- I agree with making the Mission & Vision page a "homepage" in a sense. I added some blocks there that we can use to link people through the docs. Would you be able to help me put one next to the other? ideally its a 2x2 grid vs. a 1x1. Same for the Radicle and Drips blocks.

Shelby and I are going to work on the copy more today, then I think we we'll have a cleaner version to work with.

joelhans commented on docs/community/README.md:
```diff
@@ -1,90 +1,71 @@
 
 <Grid>
   <Button
     style={{gridColumn: '1 / span 12'}}
```
```suggestion

```

joelhans commented on docs/community/README.md:
```diff
@@ -1,90 +1,71 @@
-    href="https://discord.gg/dZK4TxaU2v"
-    title="Discord"
-    cta="discord.com"
+    style={{gridColumn: '1 / span 12'}}
```
```suggestion

```

joelhans commented on docs/community/README.md:
```diff
@@ -1,90 +1,71 @@
-    href="https://radicle.community/"
-    title="Discourse"
-    cta="radicle.community"
+    style={{gridColumn: '1 / span 12'}}
```
```suggestion

```

joelhans commented on docs/community/README.md:
```diff
@@ -1,90 +1,71 @@
-    href="https://matrix.radicle.community/"
-    title="Matrix"
-    cta="matrix.radicle.community"
+    style={{gridColumn: '1 / span 12'}}
```
```suggestion

```

joelhans commented on docs/community/README.md:
```diff
@@ -1,90 +1,71 @@
-    href="https://radicle.beehiiv.com/"
-    title="Radicle Digest"
-    cta="Subscribe"
+    style={{gridColumn: '1 / span 12'}}
```
```suggestion

```

joelhans commented on docs/community/README.md:
```diff
@@ -1,90 +1,71 @@
-    href="http://eepurl.com/hhHxMX"
-    title="Radicle Governance Updates"
-    cta="Subscribe"
+    style={{gridColumn: '1 / span 12'}}
```
```suggestion

```

joelhans reviewed (commented):
Hey @abbey-titcomb! To make the grids be 2x2 and 2x1, you need to remove the `style={{gridColumn: '1 / span 12'}}` line from each `<Button>` element. That style overrides the default, which is to split into 2 columns at desktop and 1 on mobile.

I added those suggestions and you can commit them that way or go through your code to do the same thing!

shelbeeee reviewed (approved):
looks good - let's see how she runs!
